vcpkg.json

```diff
@@ -8,6 +8,9 @@
         "http2"
       ]
     },
+    {
+      "name": "libproxy"
```

Comment:
Let's keep this list and the overrides list below alphabetical. And since we aren't providing options here, you can just list "libproxy" as a string, not an object (see "libjxl" below as an example).

Reply:
Thanks, I made the changes accordingly.